sycl/CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ file(COPY ${OpenCL_INCLUDE_DIR}/CL
 
 # Include OpenCL Headers into final bundle.
 install(DIRECTORY ${OpenCL_INCLUDE_DIR}/CL
-        DESTINATION ${SYCL_INCLUDE_DEPLOY_DIR}/sycl
```

Comment:
@npmiller, can you please remove initialization of `SYCL_INCLUDE_DEPLOY_DIR` variable and replace it with `${CMAKE_INSTALL_PREFIX}/${SYCL_INCLUDE_DIR}` in `sycl/test/CMakeLists.txt`?

Reply:
Sure, patch updated!